diagnostic.md

```diff
@@ -93,5 +98,6 @@ var teamMember = {
 
 What is the value of `teamMember['hobby']`?  Of `teamMember['name']`?  Of
 `teamMember['name']['surname']`?
-
-_Replace this text with your answer._
+1) exercise
+2) Jason Weeks
```

Comment:
teamMember['name'] would actually return the object that is store under the key of 'name': {given: 'Jason', surname: 'Weeks'}
Computers have no ability to interpret our prompts, so it would not be able to combine value of 'given' and 'surname' without explicit direction to do so.